### PR TITLE
Update instructions for sim installation

### PR DESCRIPTION
**Pull request by keke-220**
## Motivation and Context
1. The previous Sim README was unclear.
2. The previous main README file contained redundant and inconsistent instructions for Sim installation. 

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist:

- [x] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] I ran `tests/hw_manual_test.py` on hardware and observed that robot behavior looks normal.

### CODE REVIEW

**Review by cpaxton — COMMENTED** [review]
One minor change - instead of listing deps add them to yaml

File: src/home_robot_sim/README.md [hunk 1]
```diff
@@ -17,14 +27,36 @@ git submodule update --init --recursive
...
+cd -
 pip install -e src/third_party/habitat-lab/habitat-lab
 pip install -e src/third_party/habitat-lab/habitat-baselines
-
 python -m pip install "git+https://github.com/facebookresearch/pytorch3d.git"
+
+conda install -c pytorch torchvision
+conda install -c anaconda scikit-image
+conda install pybind11 -c conda-forge
+pip install sophuspy
```
[the remark this answers is not in the document]
Comment (keke-220) [reply src/home_robot_sim/README.md]: It seems like those dependencies have already been added in home-robot/env.yaml, didn't notice them before. Removed these lines.

**Comment by keke-220** [comment]
> One minor change - instead of listing deps add them to yaml

Fixed

**Review by cpaxton — APPROVED** [review]
LGTM

**Pull request merged**